### DISCUSSION TITLE
Does it support 2.1?

### DISCUSSION THREAD

**Discussion by githubUser01946** [Mikubill/sd-webui-controlnet#438] [Q&A]
I have noticed that [cldm_v21.yaml](https://huggingface.co/webui/ControlNet-modules-safetensors/blob/main/cldm_v21.yaml) was uploaded a few days ago, does this mean that 2.1x is now supported with ControlNet?

**Comment by MattRM2** [comment]
Yes, it'll be great to know if controlnet is now compatible with the SD 2.1 compatible trained models.

**Comment by tsound97** [comment]
You may find some 2.1 ControlNet models trained by thibaud here:
https://huggingface.co/thibaud/controlnet-sd21/tree/main

**Reply by MattRM2** [reply]
Hi, Thank you very much, I'll try it ;)
